projects/npm-tools/packages/npm-scripts/package.json

```diff
@@ -50,6 +50,7 @@
 		"liferay-npm-bridge-generator": "2.24.2",
 		"liferay-npm-bundler": "2.24.2",
 		"liferay-npm-bundler-loader-css-loader": "2.24.2",
+		"liferay-npm-bundler-loader-json-loader": "2.24.2",
```

Comment:
Isn't there a newer version? I see this in the yarn.lock `liferay-npm-bundler-loader-json-loader@^2.27.0:`